db/migrate/20230605171154_create_hikes.rb

```diff
@@ -0,0 +1,18 @@
+class CreateHikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :hikes do |t|
+      t.string :name
+      t.string :length
+      t.text :description
+      t.string :city
+      t.string :region
+      t.string :country
+      t.string :lat
+      t.string :lon
+      t.string :difficulty
```

Comment:
Things such as difficulty or other categories are best served using Rails enum api. The datatype would be decimal but I can show you how it makes sense after you dress up the model properly.

Reply:
changed the model, but need to look more in to implementing Rails enum to update seeds data